tools/topology/topology2/platform/intel/bt-default.conf

```diff
@@ -8,11 +8,11 @@ Define {
 	BT_MCLK			38400000
 	BT_NAME			"SSP2-BT"
 
-	BT_PB_HOST_PIPELINE_ID	7
-	BT_PB_DAI_PIPELINE_ID	8
-	BT_PB_HOST_PIPELINE_SINK "copier.SSP.8.1"
-	BT_PB_DAI_PIPELINE_SRC "copier.host.7.1"
-	BT_PB_PIPELINE_STREAM_NAME "copier.SSP.8.1"
+	BT_PB_HOST_PIPELINE_ID	9
+	BT_PB_DAI_PIPELINE_ID	10
+	BT_PB_HOST_PIPELINE_SINK "copier.SSP.10.1"
+	BT_PB_DAI_PIPELINE_SRC "copier.host.9.1"
+	BT_PB_PIPELINE_STREAM_NAME "copier.SSP.10.1"
```

Comment:
@yongzhi1 These are in bt-default.conf, you can override these values in the topology file. Wouldn't that be enough?

Reply:
Ack, will update them in cavs-rt5682.conf instead, thanks!!